### PR TITLE
docs: translating

### PR DESCRIPTION
**Pull request by memtech3**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered docs/spec/dark.zh-CN.md](https://github.com/memtech3/ant-design/blob/patch-3/docs/spec/dark.zh-CN.md)

### CODE REVIEW

File: docs/spec/dark.en-US.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+2. **Consistency of Information**
+
+The information content in the dark mode needs to be consistent with the light mode, and the initialization hierarchical relationship should not be broken.
+
+## Color
+
+In the application of colors, we are based on 12 sets of basic swatches and combine longer rule processing to make colors better blend under different environmental colors.
+
+### Basic Swatches
```
Comment (afc163) [inline docs/spec/dark.en-US.md]: I prefer `Palette`.

File: docs/spec/dark.en-US.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+```__react
+import ColorPalettes from '../../site/theme/template/Color/ColorPalettes';
+
+ReactDOM.render(<ColorPalettes dark />, mountNode);
+```
+
+### Swatch Generation Tool
```
Comment (afc163) [inline docs/spec/dark.en-US.md]:
```suggestion
### Palette Generator
```

File: docs/spec/dark.en-US.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+---
+category: Global Styles
+order: 6
+title: Dark Mode
+---
+
+
```
Comment (afc163) [inline docs/spec/dark.en-US.md]: extra blank line.

File: docs/spec/dark.en-US.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Dark mode is a theme where all UI elements are darkened.
+
+## Applications of dark mode
```
Comment (afc163) [inline docs/spec/dark.en-US.md]:
```suggestion
## When to use
```

File: docs/spec/dark.en-US.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+### Color Pallete
+
+```__react
+import ColorPalettes from '../../site/theme/template/Color/ColorPalettes';
+
+ReactDOM.render(<ColorPalettes dark />, mountNode);
+```
+
+### Color Pallete Generation Tool
```
Comment (afc163) [inline docs/spec/dark.en-US.md]: Color Pallete Generator

**Comment by memtech3** [comment]
how is it now?

**Comment by memtech3** [comment]
can you help me pass the semantic pull request check? I am not familiar with semantic naming

**Comment by afc163** [comment]
Great work! Thx! @memtech3

**Pull request merged**